src/types/Deliverables.ts

```diff
@@ -0,0 +1,52 @@
+// TODO these will come from generated types
```

Comment:
@nickgraz I moved the mock types in here from services, just so we can touch fewer files later to update the import path.